developer/src/kmc-ldml/test/fixtures/sections/tran/fail-IllegalTransformAsterisk-1.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<keyboard3 xmlns="https://schemas.unicode.org/cldr/45/keyboard3" locale="mt" conformsTo="45">
+  <info name="dollarsign-fail-2" />
+  <keys />
+  <transforms type="simple">
+    <transformGroup>
+      <transform from="xq\\*" /> <!-- illegal -->
```

Comment:
Why is this illegal? 
compared with complier comment
'**Hint**: Use `\\*` to match a literal asterisk.'

Reply:
The `\\` is escaped in the compiler comment... `\*` is the actual string that the keyboard developer will see. So `\\*` is illegal because it allows an unescaped `*` in the regex, which we don't permit.